R/utilities-path.R

```diff
@@ -0,0 +1,32 @@
+
+#' Convert a path defined as string to a path array
+#'
+#' @param path A string representation of a path
+#'
+#' @return An array containing one element for each path entry
+#' @examples
+#'
+#' array <- toPathArray("Organism|Organ|Liver")
+#' @export
+toPathArray <- function(path) {
+  if (!is.character(path)) {

```

Comment:
@PavelBal As you can see. The same code is repeated over and over again
Havign a one liner validateIsString or something like that would be beneficial